app/src/test/java/com/sdp/movemeet/Activity/ActivityTest.java

```diff
@@ -37,7 +37,8 @@ public static Activity createFakeActivity() {
                 DUMMY_DATE,
                 DUMMY_DURATION,
                 DUMMY_SPORT,
-                DUMMY_ADDRESS
+                DUMMY_ADDRESS,
+                DUMMY_DATE
```

Comment:
I feel like using DATE for both the creation date and activity date might cause conflicts or at the very least confusion.